tests/mixed-source/ios/MixedSourceFramework/Logger/LoggerProtocol.h

```diff
@@ -0,0 +1,5 @@
+@import Foundation;
+
+@protocol LoggerProtocol <NSObject>
+
```

Comment:
why not just copy `@objc public func swiftLog(_ message: String)` to this one so `SwiftLogger ` actually  implements this protocol?

Reply:
Because it doesn't matter whats in the protocol, and `SwiftLogger` does indeed conform to this protocol (because there are no requirements). The conformance matters because it forces the generated swift header to have an import statement for the current module